javascript/extractor/tests/es2018/input/asyncIter.js

```diff
@@ -0,0 +1,6 @@
+async function* asyncFn() {}
+
+class C {
+  async *asyncMeth() {}
+  async *[Symbol.asyncIterator]() {}
+}
```

Comment:
Can you add a test with an async method in an object literal (if such a thing is valid)?

Reply:
Done.